benchmarks/inference_throughput/cloud-api/README.md

```diff
@@ -0,0 +1,24 @@
+# Llama-Cloud-API-Benchmark
+This folder contains code to run inference benchmark for Llama 2 models on cloud API with popular cloud service providers. The benchmark will focus on overall inference **throughput** for querying the API endpoint for output generation with different level of concurrent requests. Remember that to send queries to the API endpoint, you are required to acquire subscriptions with the cloud service providers and there will be a fee associated with it.  
+
```

Comment:
Shall we add a disclaimer that this is not representative of Azure performance and mostly a script to give a sense of the API or something along that line to clarify that its not fully representative of Azure perf.

Reply:
Good call. I will add a disclaimer here